### DISCUSSION TITLE
Should I use triple slash reference when using @eslint-types/typescript-eslint or something else?

### DISCUSSION THREAD

**Discussion by socker210** [Shinigami92/eslint-define-config#292] [Q&A] [answered]
Hi, I found awesome this package yesterday and I am currently using this package now.

When I was setup eslint configs with this, I found guide on docs:

```typescript
// @ts-check
const { defineConfig } = require('eslint-define-config');

/// <reference types="@eslint-types/typescript-eslint" />

module.exports = defineConfig({
  root: true,
  rules: {
    // rules...
  },
});
```

When I remove triple slash reference I can infer @eslint-types/typescript-eslint types. So my question is that should i write triple slash reference on config file? Maybe this question is based on typescript.

Thank you.

**Comment by Shinigami92** [comment, marked as answer]
Yes, currently this is the correct way

**Reply by socker210** [reply]
I updated my .eslintrc.cjs and tsconfig.json files:

```javascript
// @ts-check
/// <reference types="@eslint-types/typescript-eslint" />
/// <reference types="@eslint-types/import" />

const { defineConfig } = require('eslint-define-config');

module.exports = defineConfig({ ... });
```

```json
{
  "compilerOptions": {
    ...
  },
  "include": [
    ".eslintrc.cjs",
    ...
  ],
  "exclude": ["node_modules"]
}
```

and it works with type checking command.
